docs/tables/alicloud_action_trail.md

```diff
@@ -0,0 +1,63 @@
+# Table: alicloud_action_trail
+
+Alibaba Cloud ActionTrail is a service that monitors and records the actions of your Alibaba Cloud account, including the access to and use of cloud products and services through the Alibaba Cloud console, API operations, and SDKs. ActionTrail records these actions as events. You can download these events from the ActionTrail console or configure ActionTrail to deliver these events to Log Service Logstores or Object Storage Service (OSS) buckets. Then, you can perform behavior analysis, security analysis, resource change tracking, and compliance auditing based on the events.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  home_region,
+  event_rw,
+  status,
+  trail_region
+from
+  alicloud_action_trail;
+```
+
+### List trails which are Enabled
+
+```sql
+select
+  name,
+  home_region,
+  event_rw,
+  status,
+  trail_region
+from
+  alicloud_action_trail
+where
+  status = 'Enable';
+```
+
+### List trails which are older than 60 days
+
+```sql
+select
+  name,
+  home_region,
+  event_rw,
+  status,
+  trail_region
+from
+  alicloud_action_trail
+where
+  create_time <= (current_date - interval '60' day);
+```
+
+### List trails which are not multi-account trail
+
+```sql
+select
+  name,
+  home_region,
+  is_organization_trail,
+  status,
+  trail_region
+from
+  alicloud_action_trail
+where
+  not is_organization_trail;
+```
```

Comment:
@bigdatasourav can we have the list of trails where the Message Service (MNS) topic is associated ? 
i.e. is ir mandatory for any trail to have MNS topic ? I guess No.

Reply:
As discussed with Lalit , removed the column for now.